code_to_gast/expression/py_expression.py

```diff
@@ -33,10 +33,12 @@ def attribute_to_gast(node):
     if node.attr == "append":
         gast["type"] = "builtInAttribute"
         gast["id"] = "appendStatement"
+        return gast
     elif node.attr == "pop":
```

Comment:
Still have the `else` blocks!

Reply:
Wait what do you mean?

Reply:
If you return in the `if` block, there is no need to use `elif`, you can just use another `if`. That's where the benefit is, since you don't have to keep the previous `if` context in mind.

Reply:
I see okay.